examples/common/apps.ts

```diff
@@ -7,6 +7,7 @@ const apps = [
   "vercel-blog-starter",
   "vercel-commerce",
   "ssg-app",
+  "instrumentation-app",
```

Comment:
Is there a particular reason to add an app instead of re-using i.e. the playground?

Reply:
Oh probably because this is in Next15...

Does instrumentation work on Next14?

Maybe we should have a playground14 and a playground15?

Reply:
done